pkg/connector/msgconv/mentions.go

```diff
@@ -51,15 +51,21 @@ var (
 	META_MONOSPACE_BLOCK_REGEX = regexp.MustCompile("```([^`]+)```")
 )
 
-func (mc *MessageConverter) metaToMatrixText(ctx context.Context, text string, rawMentions *socket.MentionData, portal *bridgev2.Portal) (content *event.MessageEventContent) {
+func (mc *MessageConverter) MetaToMatrixText(ctx context.Context, text string, rawMentions *socket.MentionData, portal *bridgev2.Portal) (content *event.MessageEventContent) {
 	content = &event.MessageEventContent{
 		MsgType:  event.MsgText,
 		Body:     text,
 		Mentions: &event.Mentions{},
 	}
-	mentions, err := rawMentions.Parse()
-	if err != nil {
-		zerolog.Ctx(ctx).Err(err).Msg("Failed to parse mentions")
+
+	var mentions *socket.Mentions
+
+	if rawMentions != nil {
+		mentionsP, err := rawMentions.Parse()
+		if err != nil {
+			zerolog.Ctx(ctx).Err(err).Msg("Failed to parse mentions")
+		}
+		mentions = &mentionsP
```

Comment:
It would be easier to change `MentionData.Parse` to return nil if the input is nil than add a whole if like this